lib/spack/spack/build_systems/sip.py

```diff
@@ -41,32 +41,10 @@ class SIPPackage(PackageBase):
 
     depends_on('qt')
 
-    resource(name='sip',
-             url='https://www.riverbankcomputing.com/static/Downloads/sip/4.19.18/sip-4.19.18.tar.gz',
-             sha256='c0bd863800ed9b15dcad477c4017cdb73fa805c25908b0240564add74d697e1e',
-             destination='.')
```

Comment:
If we remove this resource, you need to add back the dependency on `py-sip`

Reply:
and remove py-sip dependency from PyQt?

Reply:
PyQt will still have a `py-sip` dependency because the `module=...` variant is package-specific. Every SipPackage needs a sip dependency, but some need a specific sip dependency.